crates/resolver/examples/multithreaded_runtime.rs

```diff
@@ -8,7 +8,8 @@ fn main() {
     use tokio::runtime::Runtime;
     use trust_dns_resolver::{TokioAsyncResolver, TokioHandle};
 
-    env_logger::init();
+    let subscriber = tracing_subscriber::FmtSubscriber::builder().finish();
+    tracing::subscriber::set_global_default(subscriber).expect("setting default subscriber failed");
```

Comment:
nit/tioli: could be simplified to just
```suggestion
    tracing_subscriber::fmt::init();
```
which does the same thing as this code